src/paint.js

```diff
@@ -8,7 +8,10 @@ function setupPaint({ $, qlik }) {
 
   return function ($element, layout) {
     // Call irregularUtils to page the data for > 10000 points
-    const maxPages = qlik.navigation.getMode() === "analysis" ? 10 : 1;
+    const maxPages = this.inAnalysisState() ? 10 : 1;
```

Comment:
Seems like there was another commit with changes with some of the same lines: 715f9ee But instead using `this.inEditState()`. Can we overwrite those changes with these @Caele ?

Reply:
yes you can ignore that one